tests/unit_tests/test_invalid_call.py

```diff
@@ -35,20 +24,25 @@ def bad_key():
 """
 
 
-class TestBuild:
-    @pytest.fixture
-    def file(self, tmpdir):
-        tmpdir.mkdir("tasks").join("main.py").write(TEST_FILE)
-        tmpdir.join("tasks/requirements.txt").write("magniv")
-        return str(tmpdir.join("tasks"))
+@pytest.fixture
+def file(tmpdir):
```

Comment:
Feels like repeated code. Can we make a test class or something so there is less boilerplate in new tests?

Reply:
If there is a reason this code needs to be run in this file, you can maybe make a Class with a method stub that contains the comment so at the very least the docstring doesn't need to be copied and pasted

Reply:
Good call, I can condense this